newIDE/app/src/MainFrame/EditorContainers/AskAi/AiRequestChat.js

```diff
@@ -0,0 +1,411 @@
+        if (scrollViewRef.current) {
+          scrollViewRef.current.scrollToBottom({
+            behavior: 'smooth',
+          });
+        }
+      },
+    }));
+
+    const { isMobile } = useResponsiveWindowSize();
+
+    const quotaOrCreditsExplanation = !quota ? null /* User is probably not even logged in. */ : !quota.limitReached ? (
+      quota.current < 1 || increaseQuotaOffering === 'subscribe' ? null : (
+        <Trans>
+          You still have {quota.max - quota.current} free questions thanks to
+          your membership.
+        </Trans>
+      )
+    ) : aiRequestPriceInCredits ? (
+      availableCredits ? (
+        <Trans>
+          Use an AI request for <b>{aiRequestPriceInCredits} credits</b> – you
+          have {availableCredits} credits.
+        </Trans>
+      ) : (
+        <Trans>
+          Use an AI request for <b>{aiRequestPriceInCredits} credits.</b>
+        </Trans>
+      )
+    ) : null;
+
+    const subscriptionBanner =
+      quota && quota.limitReached && increaseQuotaOffering !== 'none' ? (
+        <GetSubscriptionCard
+          subscriptionDialogOpeningReason={
+            increaseQuotaOffering === 'subscribe'
+              ? 'AI requests (subscribe)'
+              : 'AI requests (upgrade)'
+          }
+          label={
+            increaseQuotaOffering === 'subscribe' ? (
+              <Trans>Get GDevelop premium</Trans>
+            ) : (
+              <Trans>Upgrade</Trans>
+            )
+          }
+          recommendedPlanIdIfNoSubscription="gdevelop_gold"
+        >
+          <Line>
+            <Column noMargin>
+              <Text noMargin>
+                {increaseQuotaOffering === 'subscribe' ? (
+                  <Trans>
+                    Get more free AI requests with a GDevelop premium plan.
+                  </Trans>
+                ) : (
+                  <Trans>
+                    Upgrade to another premium plan to get more free AI
+                    requests.
+                  </Trans>
+                )}
+              </Text>
+            </Column>
+          </Line>
+        </GetSubscriptionCard>
+      ) : null;
+
+    const errorOrQuotaOrCreditsExplanation = (
+      <Text size="body2" color={lastSendError ? 'error' : 'secondary'}>
+        {lastSendError ? (
+          <Trans>
+            An error happened when sending your request, please try again.
+          </Trans>
+        ) : (
+          quotaOrCreditsExplanation
+        )}
+      </Text>
+    );
+
+    if (!aiRequest) {
+      const disclaimer = errorOrQuotaOrCreditsExplanation || (
+        <Text size="body2" color="secondary">
+          <Trans>
+            The AI will answer according to your game project. Always verify AI
+            answers before applying them.
+          </Trans>
+        </Text>
+      );
+
+      return (
+        <div className={classes.newChatContainer}>
+          <Line noMargin justifyContent="center">
+            <RobotIcon rotating size={40} />
```

Comment:
I would probably make the logo rotate only when it's thinking (otherwise keep it static)